requirements/dev.txt

```diff
@@ -12,41 +12,35 @@ asgiref==3.7.2
     # via
     #   -r requirements/quality.txt
     #   django
-astroid==2.15.6
+astroid==2.15.8
     # via
     #   -r requirements/quality.txt
     #   pylint
     #   pylint-celery
-bleach==6.0.0
+boto3==1.28.62
```

Comment:
There is still a boto3 requirement, but it's an older version. Is it working?

Reply:
I think that lib is being used by the xblock library but not by us directly. Either way, since it's not in the `base.in` when installing the library it won't have any effect because it's not a requirement anymore